src/services/api.js

```diff
@@ -114,28 +114,6 @@ async function getDownload(props, file) {
     })
 }
 

```

Comment:
There was an existing call in here.  What wsa the reason for moving it?

Reply:
Not sure what's being quoted here.